src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerAssemblyLoader.cs

```diff
@@ -13,6 +13,15 @@
 
 namespace Microsoft.CodeAnalysis
 {
+    internal interface IAnalyzerAssemblyLoaderInternal : IAnalyzerAssemblyLoader
```

Comment:
`IAnalyzerAssemblyLoader` is a `public` API and did not want to create new `public` API for this hence went with an `internal` API instead.

Reply:
We don't expect external users (at least those who lack IVT) to implement IAnalyzerAssemblyLoader, right?

Reply:
Correct. This is only used in the compiler server where external teams cannot provide implementations.